WeakAurasOptions/WeakAurasOptions.lua

```diff
@@ -1295,8 +1295,11 @@ function WeakAuras.DeleteOption(data, massDelete)
 
   frame:ClearPicks();
   WeakAuras.Delete(data);
-  frame.buttonsScroll:DeleteChild(displayButtons[id]);
-  displayButtons[id] = nil;
+
+  if(displayButtons[id])then
```

Comment:
I don't understand how this can happen.

Reply:
It happens when you import a update WA if you never opened WA options before. As buttons are created in "async" way.